easybuild/easyconfigs/f/freebayes/freebayes-1.3.2-foss-2018b.eb

```diff
@@ -0,0 +1,42 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'MakeCp'
+
+name = 'freebayes'
+version = '1.3.2'
+
+homepage = 'https://github.com/ekg/%(name)s'
+description = "Bayesian haplotype-based polymorphism discovery and genotyping."
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+# To download freebayes, use git with the --recursive option.
+# This is required in order to download all nested git submodules for external repositories.
+# Afterwards pack everything into a tarball:
+#   git clone --recursive -b v1.3.2 git://github.com/ekg/freebayes.git
+#   tar czf freebayes-1.3.2.tar.gz freebayes
+# no checksum for freebayes-1.3.2.tar.gz since it's created from git repo
+sources = ['%(name)s-%(version)s.tar.gz']
```

Comment:
@SethosII to work around the lack of checksums, please add:
`checksums = [None]`

Reply:
@verdurin I changed it.

Reply:
@SethosII A better option would be to let EasyBuild download the sources via `git clone --recursive`, see https://docs.easybuild.io/en/latest/Writing_easyconfig_files.html#common-easyconfig-param-sources-git-config

Reply:
@boegel Thanks for the hint, I changed it.